src/config.c

```diff
@@ -3276,6 +3276,7 @@ standardConfig static_configs[] = {
     createIntConfig("timeout", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.maxidletime, 0, INTEGER_CONFIG, NULL, NULL), /* Default client timeout: infinite */
     createIntConfig("replica-announce-port", "slave-announce-port", MODIFIABLE_CONFIG, 0, 65535, server.replica_announce_port, 0, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("tcp-backlog", NULL, IMMUTABLE_CONFIG, 0, INT_MAX, server.tcp_backlog, 511, INTEGER_CONFIG, NULL, NULL), /* TCP listen backlog. */
+    createBoolConfig("mptcp", NULL, IMMUTABLE_CONFIG, server.mptcp, 0, NULL, NULL),                                          /* Multipath TCP. */
```

Comment:
Linked to my previous comment:

```suggestion
    createBoolConfig("mptcp", NULL, IMMUTABLE_CONFIG, server.mptcp, 1, NULL, NULL),                                          /* Multipath TCP. */
```

Reply:
Let's use a validation function:

```suggestion
    createBoolConfig("mptcp", NULL, IMMUTABLE_CONFIG, server.mptcp, 0, isValidMptcp, NULL),                                  /* Multipath TCP. */
```

This function can be defined like this:

```C
static int isValidMptcp(int val, const char **err) {
#ifndef IPPROTO_MPTCP
    if (val) {
        *err = "MPTCP is not supported on this platform");
        return 0;
    }
#else
    UNUSED(val);
    UNUSED(err);
#endif
    return 1;
}
```